### ISSUE TITLE
CTA / Content Block Simple - downloads a file, open external link information is not conveyed for the screen reader user.

### ISSUE THREAD

**Issue by Praveen-sr** [carbon-design-system/carbon-for-ibm-dotcom#4142]
## Detailed description
On JAWS FF &  iPhone 

Issue: There is an icon near these links which indicates it downloads a file, open external link  but, the same information is not conveyed  for the screen reader user.

Note: the same issue we found in react staging as well.

Expected: 
The information conveyed visually should be available for the screen reader user as well. Anyone of the following methods can be used to make it screen reader accessible.
1. provide the information in the link text. 
2. title attribute. or 
3. as an off screen text
a generic example: 
1. feed back form opens in new window
2. products opens external link

## Additional information

![image.png](https://images.zenhubusercontent.com/5e21a7b470118a498429a20e/36b652e9-78ff-4980-b3ed-0713af7fb9b2)
